packages/docusaurus-init/templates/classic/docusaurus.config.js

```diff
@@ -75,6 +79,10 @@ module.exports = {
       {
         docs: {
           sidebarPath: require.resolve('./sidebars.js'),
+          editUrl:
+            'https://github.com/facebook/docusaurus/edit/master/website/',
+          showLastUpdateAuthor: true,
```

Comment:
I think maybe we shouldnt the last updated time & author by default. This slows build down (like a LOT) and I think its nicer to give them an impression that "Wow, docusaurus 2 is fast". For newly init site (and lot of our user) does not use git yet, so they're going to see "Sorry, this requires git" error also. Only if they need it they will need to enable it.

and the editUrl is also pointing to the wrong repo.

Reply:
Ok I'll remove the last updated time and author display.

> and the editUrl is also pointing to the wrong repo.

This is intended to give them the right default value (if they create a `website` folder at the root of their repo). We don't know their repo so it's never going to be right.